Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -76,6 +76,14 @@ public static void Start()
             HttpApplication.RegisterModule(typeof(NinjectHttpModule));
             HttpApplication.RegisterModule(typeof(TraceModule));
             _bootstrapper.Initialize(CreateKernel);
+
+            // if ProgramData already have the msbuild15/state.json,do nothing
+            // else copy state.json for msbuild15 discovery purpose
+            string dest = $"{VfsSpecialFolders.ProgramDataPath}\\Microsoft\\VisualStudio\\Packages\\_Instances";
```

Comment:
Don't run this on Linux. I don't think it'll break anything, but better to be explicit.